geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template

```diff
@@ -58,8 +58,8 @@
     -->
 
     <property>
-        <name>geomesa.batchwriter.latency.millis</name>
-        <value>60000</value>
+        <name>geomesa.batchwriter.latency</name>
+        <value>60 seconds</value>
```

Comment:
@jahhulbert-ccri thoughts about changing things here in the geomesa-site.xml?

Reply:
Do we need to note that in the upgrade guide?

Reply:
I'll add a note about geomesa-site.xml, thanks

Reply:
added